Assets/Mirage/Runtime/Entity/INetworkObject.cs

```diff
@@ -0,0 +1,10 @@
+namespace Mirage.Entity
+{
+    public interface INetworkObject
```

Comment:
Dont need both `INetworkObject` and `INetworkIdentity` since there should only be 1 Identity per Object, they can just be 1 interface